src/Arcus.Scripting.ApiManagement/Arcus.Scripting.ApiManagement.psd1

```diff
@@ -48,7 +48,7 @@ Description = 'Scripts related to Azure API Management'
 # ProcessorArchitecture = ''
 
 # Modules that must be imported into the global environment prior to importing this module
-RequiredModules = @('Az.ApiManagement')
+RequiredModules = @("Az.ApiManagement")

```

Comment:
What if we try this?
```suggestion
RequiredModules = @(@{ModuleName = 'Az.ApiManagement'; ModuleVersion = '2.0.1'; })
```

Reply:
However, I would add the info mentioned above as well when testing, maybe it's because of the combination (min version + module)